plugins/modules/mount.py

```diff
@@ -171,10 +174,12 @@ def is_fspath_mounted(module, mount_dir, mount_over_dir):
     """
     global result
 
-    if mount_dir:
-        cmd = "lsfs -l %s" % mount_dir
-    elif mount_over_dir:
+    if (mount_dir and mount_over_dir) or mount_over_dir:
```

Comment:
can we simplify the check to "if mount_over_dir"? looks like no need to check mount_dir here

Reply:
you are correct. this is some leftover from previous logic. the order was different before.
i wanted to make sure that if both `mount_dir` and `mount_over_dir` was specified, i would
check for `mount_over_dir`, but since i changed the order to check for `mount_over_dir` first
then there is not need for that extra condition.
okay, i will simplify to
```
if mount_over_dir:
 ...
elif mount_dir:
 ...
else:
 ...
```